### PR TITLE
Various fingerprint updates

### PR DESCRIPTION
**Pull request by hdm**
## Description

This is a collection of fingerprint updates and cleanups encountered while testing Recog.

## Motivation and Context

This PR improves the coverage and consistency of fingerprints.

## How Has This Been Tested?

Real-world testing and rspec

## Types of changes
- New fingerprints
- Removal of trailing whitespace
- Small changes to make some fingerprints more consistent

## Checklist:
- [x] I have updated the documentation accordingly (or changes are not required).
- [x] I have added tests to cover my changes (or new tests are not required).
- [x] All new and existing tests passed.

### CODE REVIEW

File: xml/telnet_banners.xml [hunk 1]
```diff
@@ -1573,4 +1576,26 @@
...
+      UHJlY2lzZS9SVENTIHYyLjkwLjAwIFRlbG5ldCBzZXJ2ZXIKCgpTZXJ2aWNlIFBvcnQgTWFuYWdlciBBY3RpdmUKCjxFc2M+IEVuZHMgU2Vzc2lvbgoKroot
+    </example>
+    <param pos="0" name="hw.device" value="Power device"/>
+    <param pos="0" name="hw.vendor" value="Liebert"/>
+    <param pos="0" name="os.device" value="Power device"/>
+    <param pos="0" name="os.vendor" value="Liebert"/>
+    <param pos="1" name="os.version"/>
+  </fingerprint>
+
+
```
Comment (rkirk-r7) [inline xml/telnet_banners.xml:1597]: extra whitespace?

Comment (hdm) [reply xml/telnet_banners.xml]: Thanks!

**Comment by tsellers-r7** [comment]
Thanks for the contributions @hdm !

**Comment by hdm** [comment]
Thanks yall!

**Pull request merged**